openfisca_core/entities.py

```diff
@@ -471,6 +491,7 @@ def value_from_first_person(self, array):
 
     # Projection entity -> person(s)
 
+    @projectable
```

Comment:
Does project need to be projectable ?

Reply:
No, it doesn't work, and would not make a lot of sense 👍